docs/docs/releases.md

```diff
@@ -119,21 +119,21 @@ make
 
 ### Binaries
 
-Official binaries can be found on our [GitHub Releases](https://github.com/pomerium/pomerium/releases) page.
+Official binaries can be found on our [GitHub Releases](https://github.com/pomerium/cli/releases) page.
 
 ```shell
 ARCH=[your arch]
 OS=[your os]
 VERSION=[desired version]
-curl -L https://github.com/pomerium/pomerium/releases/download/${VERSION}/pomerium-cli-${OS}-${ARCH}.tar.gz \
+curl -L https://github.com/pomerium/cli/releases/download/${VERSION}/pomerium-cli-${OS}-${ARCH}.tar.gz \
     | tar -z -x
 ```
 
 ### Packages
 
 - Supported formats: `rpm`, `deb`
 
-Official packages can be found on our [GitHub Releases](https://github.com/pomerium/pomerium/releases) page or from [Cloudsmith](https://cloudsmith.io/~pomerium/repos/pomerium/packages/).
+Official packages can be found on our [GitHub Releases](https://github.com/pomerium/cli/releases) page or from [Cloudsmith](https://cloudsmith.io/~pomerium/repos/pomerium/packages/).
```

Comment:
nuh uh 
![image](https://user-images.githubusercontent.com/2349184/144094300-91c84dda-3320-4e88-8520-71d2c3bbbaf4.png)

Reply:
That's expected - there aren't any yet.  It will go live with v0.16.